src/views/template/topbar.html

```diff
@@ -21,14 +21,17 @@
         </li>
     </ul>
     <div class="nav navbar-nav navbar-right statuses">
-        
         <p class="navbar-text">
             <strong>{{#l}}Level{{/l}}</strong>
             <span id="level">{{ level }}</span>
         </p>
-        <p class="navbar-text hidden">
-            <strong class="text-capitalize">{{#l}}satisfaction{{/l}}</strong>
-            <span id="satisfaction">{{ config.satisfaction }}</span>%
+        <p class="navbar-text">
+            <strong>{{#1}}Score{{/1}}</strong>

```

Comment:
Is this a `1`, or an `l`? It should be an `l`.

Reply:
This explains a lot. That's the translate wrap right? Satisfaction also should be translated I guess?

Reply:
Yes. `l` for **l**anguage.